installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.adoc

```diff
@@ -0,0 +1,113 @@
+[id="installing-azure-stack-hub-user-infra"]
+= Installing a cluster on Azure Stack Hub using ARM templates
+include::modules/common-attributes.adoc[]
+:context: installing-azure-stack-hub-user-infra
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on Microsoft Azure Stack Hub by using infrastructure that you provide.
+
+Several link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/overview[Azure Resource Manager] (ARM) templates are provided to assist in completing these steps or to help model your own.
+
+[IMPORTANT]
+====
+The steps for performing a user-provisioned infrastructure installation are provided as an example only. Installing a cluster with infrastructure you provide requires knowledge of the cloud provider and the installation process of {product-title}. Several ARM templates are provided to assist in completing these steps or to help model your own. You are also free to create the required resources through other methods; the templates are just an example.
+====
+
+[id="prerequisites_installing-azure-stack-hub-user-infra"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* You xref:../../installing/installing_azure_stack_hub/installing-azure-stack-hub-account.adoc#installing-azure-stack-hub-account[configured an Azure Stack Hub account] to host the cluster.
+* You downloaded the Azure CLI and installed it on your computer. See link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest[Install the Azure CLI] in the Azure documentation. The documentation below was tested using version `2.0.80` of the Azure CLI. Azure CLI commands might perform differently based on the version you use.
+* If you use a firewall and plan to use the Telemetry service, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured the firewall to allow the sites] that your cluster requires access to.
++
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
```

Comment:
This note has been copied/pasted into every UPI assembly we have :sweat_smile:. I believe it's referring to the list linked in the prereq it's attached to. Since this is a standard prereq copied across all UPI installation assemblies, we can maybe plan a follow up for them all, if necessary.